### ISSUE TITLE
layout issues on iOS with 1.1.1

### ISSUE THREAD

**Issue by npomfret** [react-native-community/react-native-blur#109]
I've just had to revert back to 1.0.1 form 1.1.1.  It's a bit difficult to describe but I was getting all kinds of weird layout issues.  It looked a bit like the bounds of my blur view were reaching beyond the component it was in.  It resulted in other component on the screen having 'bits' of blur view in them.

Like I said difficult to describe, but here's a screen shot.  The boxes with text in on the left should be very dark, and the ones on the right very light. You can see the top right box seems to have 2 background colours, as do the bottom left 2.
 
![screen shot 2016-11-02 at 15 45 09](https://cloud.githubusercontent.com/assets/5489469/19936084/43ff9ee4-a114-11e6-88bf-03d22077c0ae.png)

I reverted back to 1.0.1 on RN0.36 and then layout problems went away.

**Comment by Kureev** [comment]
Can you show us your code? It's kinda complicated to debug based on the screenshot :)

**Comment by npomfret** [comment]
Well ultimately I'm just creating a blur view with stuff in it!  nothing complicated.  But the behaviour has changed quite significantly between 1.0.1 and 1.1.1.  I reverted and the app is displaying properly again. I don't think it's anything to do with my code and the only change I made was to the package.json.

Here's the code, but without the context it isn't going to be any use I imagine.

```
        <View ref={ref}>
          <BlurView blurType={iosBlurType} style={[listCellStyle, {minWidth: minWidth}]}>
            {messageHeader}
            {this.props.children}
            {footer}
          </BlurView>
        </View>
```
